src/styles/_mixins.scss

```diff
@@ -67,3 +67,12 @@ $global-right: if($global-text-direction == rtl, left, right);
   overflow: hidden;
   text-overflow: ellipsis;
 }
+
+@mixin hover($with-transition: false) {
+  filter: brightness(110%);
+  @if $with-transition == true {
+    transition: 0.5s filter ease;
+  } @else {
+    transition: none;
```

Comment:
It would be better to not define the else altogether, as it's not needed and might deactivate other transitions on the elements using the hover mixin

Reply:
I added it because the `a:hover` was applied to some of the components where we didn't want it to be applied (like in facets). Maybe there's a better way to handle this?

Reply:
OK, that's strange and probably an issue that it would do that in facets if we don't want it too. But I guess the transition override shouldn't bother us too much anyway in other places.
Go for it then, you can keep it